compiler/passes/codegen.cpp

```diff
@@ -360,6 +361,38 @@ compareSymbol(const void* v1, const void* v2) {
   return result;
 }
 
+static int
```

Comment:
The requirements on the comparison function are that it returns a type convertible to bool, so bool would work in the signature.